emote/nn/layers.py

```diff
@@ -174,3 +175,159 @@ def get_encoder_output_size(self):
             out_size = np.prod(out_size)
 
         return out_size
+
+
+class FullyConnectedEncoder(nn.Module):
```

Comment:
I think we've argued back and forth whether we should have encoders etc, or if Emote should prefer lower-level building blocks only.

Reply:
Maybe worth making a call here and then sticking to it. cc @singhblom

Reply:
oof. Yes, we have. I think my opinion here is actually that things that are this big should probably be in the protocol in erupt.

Reply:
Sounds very right to me :)

Reply:
Fixed!